db/schema.rb

```diff
@@ -149,7 +149,7 @@
     t.integer  "user_id",    limit: 4,                              null: false
     t.integer  "project_id", limit: 4,                              null: false
     t.string   "status",     limit: 255,        default: "pending"
-    t.text     "output",     limit: 1073741823
+    t.text     "output",     limit: 4294967295
```

Comment:
I think this is wrong / dangerous ... try to only add the changes you need (copy and the git co master -- db/schema.rb)